dxr/vcs.py

```diff
@@ -113,8 +114,9 @@ def find_previous_revisions(self, client):
         """
         last_change = {}
         for line in client.rawcommand(['previous-revisions']).splitlines():
-            path, node = line.split(':')
-            last_change[path] = node
+            if ':' in line:
+                path, node = line.split(':', 1)
+                last_change[path] = node
```

Comment:
If there is a newline in a filename, this is still going to make a hash entry. I think this fix would benefit from a test or 3.